docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1549,172 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
 command.
 
 
+.. index:: pair: command groups;statistics command group
+
+.. _`Statistics command group`:
+
+``statistics`` command group
+----------------------------
+
+The ``statistics`` command group includes commands that manage and display
+statistics on the WBEM server operations executed by pywbemcli.
+
+Statistics can be gathered in the pywbemcli client itself and also separately
+by some WBEM servers.
+
+These servers provide statistical information on the overall server
+time that can be reported to the client with each operation response.  The
+enabling/disabling of client statistics and server statistics are separate.
+
+Client statistics gathering is enabled with the pywbemcli general option
+``--timestats`` / ``-T`` including  number of operations
+executed for each operation type, operation execution time (mininum, maximum, and
+average), minimum, maximum, and average size of requests and responses, and
+the server response time if it is enabled on the server.
+
+.. index:: pair: Operation statistics;statistics commands
+
+.. _Operation statistics:
+
+There are two statistics gathering capabilities:
+
+1. pywbem gathers statistics and pywbbemcli reports them on requests and
+responses executed by pywbemcli against a WBEM server. This statistics
+gathering is enabled with the general option ``--timestats`` / ``-T``.
+
+2. Some WBEM servers can gather statistics on the requests processed by the
+server.  There are two components to this statistics gathering:
+
+  a. Gathering statistical information on requests executed against the
+  server including processing time, number of operation, sizes of request and
+  responses.  This data is gathered in the server when statistics gathering
+  is enabled and can be retrieved by the client by executing an enumerate
+  instances on the class WBEMServerResponseTime which is normally
+  maintained in the WBEM server interop namespace.
+
+  b. Server response time for each operation returned to the client with each
+  CIM-XML operation response in the header field ``WBEMServerResponseTime``.
+  Pywbem/pywbemcli includes this data in its statistics report if the server
+  returns the data.
+
+The statsitics commands are:
+
+* :ref:`Statistics reset command` - Reset statistics counts in pywbemcli.
+* :ref:`Statistics server-on command` - Enable statistics gathering in a compliant WBEM server.
+* :ref:`Statistics server-off command` - Disable statistics gathering in a compliant WBEM server.
+* :ref:`Statistics server-show command` - Display statistics counts in enabled WBEM server.
+* :ref:`Statistics show command` - Display statistics counts in pywbemcli.
```

Comment:
DONE

Reply:
NOTE: That term is not really true since they are all statisitcs on the server or server/client interaction.  The real difference is where the statistics are kept and managed.  But, for now I did not change that.